roles/DCOS.bootstrap/tasks/main.yml

```diff
@@ -115,30 +115,33 @@
     creates: "{{ download_path }}/genconf/serve/dcos_install.sh"
 
 # vvv GENERATE UPGRADE SCRIPTS vvv
-- name: generate DC/OS upgrade files
+- name: "generate DC/OS upgrade files"
```

Comment:
from the style perspective either every task should have a quoted name or none

Reply:
Good catch! The rule we tried to follow in the past was what is enforced by Ansible: double-quote whenever there is string interpolation. But yeah, also consolidating style with capitalization and such.
I'd suggest to keep that to its own PR, however.